packages/web-embed-api/src/index.ts

```diff
@@ -136,12 +136,7 @@ const FormsortWebEmbed = (
 };
 
 export { IFormsortWebEmbed, IFormsortWebEmbedConfig, IEventMap };
-export {
-  type IAnalyticsEventMap,
-  type IEventListener,
-  type IEventListenersArrayMap,
-  isSupportedEventType,
-  SupportedAnalyticsEvent,
-} from './EmbedMessagingManager'; 
+
+export { SupportedAnalyticsEvent } from '@formsort/embed-messaging-manager';
```

Comment:
@fzembow I added `SupportedAnalyticsEvent` back since we use it in other packages and formsort repo. We can remove this when we update `@formsort/embed-messaging-manager` in formsort and other packages.